src/libs/actions/Report.ts

```diff
@@ -2953,6 +3000,10 @@ function resolveActionableMentionWhisper(reportId: string, reportAction: OnyxEnt
     API.write(WRITE_COMMANDS.RESOLVE_ACTIONABLE_MENTION_WHISPER, parameters, {optimisticData, failureData});
 }
 
+function setGroupDraft(invitedUsersIDs: number[], reportName = '') {
+    Onyx.set(ONYXKEYS.NEW_GROUP, {selectedOptions: invitedUsersIDs, reportName});
```

Comment:
```suggestion
    Onyx.merge(ONYXKEYS.NEW_GROUP, {selectedOptions: invitedUsersIDs, reportName});
```

merge is preferred whenever possible